LAD-AMA-Common/metrics_ext_utils/metrics_ext_handler.py

```diff
@@ -600,6 +600,12 @@ def get_imds_values(is_lad):
         else:
             imds_url = "http://169.254.169.254/metadata/instance?api-version=2019-03-11"
 
+    message = "IMDS url to query: " + imds_url
+    if HUtilObj is not None:
+        HUtilObj.log(message)
+    else:
+        print('Info: {0}'.format(message))
```

Comment:
how frequently is this going to be logged?

Reply:
during enable call. this is needed for troubleshooting connectivity issuesthat can happen post ama install.

Reply:
need to ensure this does not cause repeated logs. see multiple incidents regarding extension.log size explosion